docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc

```diff
@@ -517,6 +518,18 @@ Bulk Loader is executing.
 specifies that the target table, which is an index, be populated with
 data from the parent table.
 
+** `REBUILD INDEXES`
++
+specifies that indexes of the target table will be updated automatically when the source table 
+is updated. 
++
+This is the default behavior of the LOAD Statement, that is, even if this option is not 
+specified, the LOAD Statement will rebuild indexes except the 
+CQD `TRAF_LOAD_ALLOW_RISKY_INDEX_MAINTENANCE` is turned *ON*. This CQD is turned *OFF* by default, 

```

Comment:
"This CQD is turned *OFF* by default, ..." Suggest making this a complete sentence. What comes after it seems to stand well on its own.

Reply:
Thanks Dave, your comment has been incorporated :)